app/src/main/java/cz/quanti/razym/rocketapp/presentation/RocketDetailViewModel.kt

```diff
@@ -51,7 +52,7 @@ class RocketDetailViewModel(
 }
 
 fun RocketData.asRocketDetail(): RocketDetail {
-    return RocketDetail(this.name, this.firstFlight, this.id, this.overview,
+    return RocketDetail(this.name, firstFlightParser.parse(this.firstFlight)!!, this.id, this.overview,
```

Comment:
tohle muze hodit vyjimku, tak by bylo dobry obalit volani toho mapperu do nejakeho runCatching, aby to neshodilo aplikaci, ale spis ukazalo error stav